src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int INITIAL_THRESHOLD = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    private static final int INCREASE_MULTIPLIER = 2;
+
+    private Node<K, V>[] map;
+    private int size;
+    private int threshold;
+
+    private static class Node<K, V> {
+
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+
+    }
+
+    public MyHashMap() {
+        map = new Node[INITIAL_CAPACITY];
+        threshold = INITIAL_THRESHOLD;
```

Comment:
```suggestion
        threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
```
then remove INITIAL_THRESHOLD